client/me/billing-history/transactions-header.jsx

```diff
@@ -249,14 +259,14 @@ const TransactionsHeader = createReactClass( {
 				</div>
 			</div>
 		);
-	},
+	}
 
-	getApps: function() {
+	getApps() {
 		return uniq( map( this.props.transactions, 'service' ) );
-	},
+	}
 
-	renderAppPicker: function( title, app, analyticsEvent ) {
-		var filter = { app: app },
+	renderAppPicker( title, app, analyticsEvent ) {
+		const filter = { app: app },
 			classes = classNames( {
```

Comment:
Want to do independent declarations as long as you're here?

Reply:
👍  d252e6781df12401f40ca9ccd9288b88adaaefba